backend/dataall/cdkproxy/stacks/policies/service_policy.py

```diff
@@ -30,88 +36,95 @@ def __init__(
+        from .sqs import SQS
+        from .ssm import SSM
         from .stepfunctions import StepFunctions
-        from .quicksight import QuickSight
-        from .cloudformation import Cloudformation
 
         policies: [aws_iam.ManagedPolicy] = [
-            # This policy covers the minumum actions required independent
-            # of the service permissions given to the group.
-            # The 'glue:GetTable', 'glue:GetPartitions' and
-            # 'lakeformation:GetDataAccess' actions are additionally
-            # required for the Worksheet/Athena feature.
+            # This policy adds some minimum actions required independent from the services enabled for the group
             aws_iam.ManagedPolicy(
                 self.stack,
                 self.id,
                 managed_policy_name=f'{self.id}-0',
                 statements=[
                     aws_iam.PolicyStatement(
+                        sid="ListActions",
+                        effect=aws_iam.Effect.ALLOW,
                         actions=[
-                            'athena:ListEngineVersions',
-                            'athena:ListDataCatalogs',
-                            'athena:ListWorkGroups',
-                            'glue:GetTable',
-                            'glue:GetPartitions',
-                            'lakeformation:GetDataAccess',
-                            'kms:Decrypt',
-                            'kms:DescribeKey',
-                            'kms:Encrypt',
-                            'kms:ReEncrypt*',
-                            'kms:GenerateDataKey*',
-                            'kms:CreateGrant',
-                            'secretsmanager:GetSecretValue',
-                            'secretsmanager:DescribeSecret',
-                            'secretsmanager:ListSecrets',
-                            'ssm:GetParametersByPath',
-                            'ssm:GetParameters',
-                            'ssm:GetParameter',
                             'ec2:Describe*',
                             'logs:Describe*',
                             'logs:Get*',
                             'logs:List*',
+                            'cloudwatch:GetMetricData',
+                            'events:ListRuleNamesByTarget',
                             'iam:list*',
                             'iam:Get*',
+                            'iam:CreatePolicy',
+                            'iam:CreateServiceLinkedRole',
                             'tag:GetResources',
-                            'tag:TagResources',
-                            'tag:UntagResources',
                             'tag:GetTagValues',
                             'tag:GetTagKeys',
                         ],
                         resources=['*'],
                     ),
                     aws_iam.PolicyStatement(
+                        sid="CreateServiceRole",
+                        actions=[
+                            'iam:CreateRole',
```

Comment:
Can't we create all roles we need and then use assume role for services?

Reply:
This one is arguable. The reason to have it is that some aws services require a service role or work easily with a service role autocreation